hip-service/Link/Patient/Constants.cs

```diff
@@ -0,0 +1,7 @@
+namespace hip_service.Link.Patient
+{
+    public static class Constants
+    {
+       public const string DateTimeFormat = "yyyy-MM-ddTHH:mm:ssZ";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4769144)